examples/SimpleDrone/helloworld.py

```diff
@@ -0,0 +1,68 @@
+'''
+helloworld.py
+This program will draw 3DR in the sky. It is designed to be run onboard solo.
+It utilizes SimpleDrone which allows for directional control using dronekit
+Default size(scale = 1.0) is 4.4x2 meters. Change the scale factor to increase size.
+Programming exercise:
+Switch y and z axis to make a cool night writing program.
+'''
+
+from dronekit import *
+from SimpleDrone import SimpleDrone
+import time
+
+SIM = False
+
+def translate(x=0, y=0, z=0):
+    scale = 1.0
+    sd.translate(x= x*scale,y=y*scale,z=z*scale, wait_for_arrival=True)
+
+print "connecting to drone..."
+if SIM:
+    vehicle = connect('127.0.0.1:14551', wait_ready=True)
+else:
+    vehicle = connect('udpout:127.0.0.1:14560', wait_ready=True)
```

Comment:
Really? That's not what we say and explain in the guide here http://dev.3dr.com/example-dronekit.html#connecting-to-solo